src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,39 +55,43 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
-        return Optional.empty();
+        try {
+            return Optional.of(LocalDate.of(dateParams[0], dateParams[1], dateParams[2]));
+        } catch (DateTimeException | IndexOutOfBoundsException e) {
+            return Optional.empty();
+        }
```

Comment:
we can catch two particular exceptions in the same catch block, but in case of any trouble we should be able to identify them - for now you can use System.out.println(), later we will learn logging

move return Optional.empty(); to the end of method